stdlib/src/utils/index.mojo

```diff
@@ -191,36 +191,26 @@ struct StaticIntTuple[size: Int](Sized, Stringable, EqualityComparable):
     """The underlying storage of the tuple value."""
 
     @always_inline
-    fn __init__() -> Self:
-        """Constructs a static int tuple of the given size.
-
-        Returns:
-            The constructed tuple.
-        """
-        return 0
+    fn __init__(inout self):
+        """Constructs a static int tuple of the given size."""
+        self = 0
 
     @always_inline
-    fn __init__(value: __mlir_type.index) -> Self:
+    fn __init__(inout self, value: __mlir_type.index):
         """Constructs a sized 1 static int tuple of given the element value.
 
         Args:
             value: The initial value.
-
-        Returns:
-            The constructed tuple.
         """
         constrained[size == 1]()
-        return Int(value)
+        self = Int(value)
```

Comment:
**Suggestion** This can just be `self = value` I think if you'd like.

Reply:
Thank you! Updated